### DISCUSSION TITLE
how to enable nfc support under fedora 37

### DISCUSSION THREAD

**Discussion by ionfan** [Yubico/libfido2#697] [Q&A]
Hello,
in this github issue it is mentioned about 3 years ago that experimental NFC-Support has been implemented for Linux (https://github.com/Yubico/libfido2/issues/247#issuecomment-749513159).
Can someone tell me how i can activate that NFC-support for libfido2 in fedora? And also does someone know if there is a chance to get that included as fully working and not “experimental”?
I holp you can help me.
Thank you in advance.
Regards

**Comment by LDVG** [comment]
Hi,

The NFC support you're referring to was [enabled by default in version 1.9.0](https://github.com/Yubico/libfido2/blob/c67dbe36592590df08e265d6347aeff231ba0470/NEWS#L57). The implementation uses the Linux NFC subsystem and require an NFC reader compatible with a kernel driver.

We also have an [experimental option to enable support for PC/SC](https://github.com/Yubico/libfido2/blob/c67dbe36592590df08e265d6347aeff231ba0470/README.adoc?plain=1#L124) you can try for readers not supported by the Linux NFC subsystem or for Windows and MacOS. For this you'd have to rebuild libfido2 from source.
